src/Training.AirBnb.Clone.Backend/Backend Project/Domain/Entities/EmailTemplate.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Backend_Project.Domain.Common;
+
+namespace Backend_Project.Domain.Entities;
+public class EmailTemplate : SoftDeletedEntity
+{
+    public string Subject { get; set; }
+    public string Body { get; set; }
+
+
+    public EmailTemplate(Guid id, string subject, string body)
+    {
```

Comment:
Id tashqaridan kirib kelishi kerak emas. Guid bo'lganligi uchun uni constructor ichida generate qilib qo'ysa bo'ladi. Agar tashqaridan id kirib keladigan bo'lsa uni validatsiya qilishga to'g'ri kelib qoladi.